versions.yaml

```diff
@@ -305,7 +305,7 @@ externals:
       package_output_dir: "OvmfX64"
     sev:
       description: "AmdSev build needed for SEV measured direct boot."
-      version: "edk2-stable202302"
+      version: "edk2-stable202402"
```

Comment:
Just one comment here.  It seems we can use stable edk2 pretty much evrywhere, as that has support for SEV / SNP / TDX.

Would it be possible to actually keep the versions of those aligned?